libraries/libfc/include/fc/time.hpp

```diff
@@ -11,7 +12,7 @@ namespace fc {
   class microseconds {
     public:
         constexpr explicit microseconds( int64_t c = 0) :_count(c){}
-        static constexpr microseconds maximum() { return microseconds(0x7fffffffffffffffll); }
+        static constexpr microseconds maximum() { return microseconds(std::numeric_limits<int64_t>::max()); }
```

Comment:
This seems like unnecessary changes for a backport. I would suggest we only do this for `main`.

Reply:
sure, will remove for the backports.